frontend/src/utils/dialogs/FarmWithdrawDialog.vue

```diff
@@ -146,16 +184,11 @@ export default class FarmWithdrawDialog extends Vue {
 			nonce,
 			deadline
 		)
-		const weth = await TokensUtil.getTokenBySymbol('WETH')
-		if (!weth) {
-			console.error("Can't find WETH")
-			return
-		}
-		const tokenAddress = this.token.address ? this.token.address : weth[0].address
+
 		const tx = await zap.zapOutToTokenWithPermit(
 			amountToZap,
 			tokenAddress,
-			0, // TODO: find a way to get a better estimate,
+			swapAmountOut,
```

Comment:
the swapAmountOut is not shown to the user before the tx right? Could that be handy?

Reply:
It's only dumped in the console, indeed.
To think about for a next release.